serverRoot/src/main/server/org/compiere/server/AdempiereServerMgr.java

```diff
@@ -557,7 +564,11 @@ public String getServerCount()
 	 */
 	public Timestamp getStartTime()
 	{
-		return m_start;
+		return startTime;
 	}	//	getStartTime
 
+	public static void main(String[] args) {

```

Comment:
Why a main method?

Reply:
Just go to run with current IDE. Maybe can be removed